public/modules/custom/grants_handler/src/Form/MessageForm.php

```diff
@@ -164,9 +165,76 @@ public function buildForm(array $form, FormStateInterface $form_state, WebformSu
       '#value' => $this->t('Send'),
     ];
 
+    $form_state->setStorage($storage);
+
     return $form;
   }
 
+  /**
+   * Validate & upload file attachment.
```

Comment:
Funktiota käytetään uploadaamiseen vaikka formissa se on kytketty validate-toiminnallisuuteen. Ehkä asiaa voisi parilla sanalla avata kommentissa miksi näin on.